content/en/blog/multiplexer/index.md

```diff
@@ -0,0 +1,326 @@
+---
+date: 2022-09-30
+title: "Ledger Multiplexer"
+linkTitle: "Ledger Multiplexer"
+description: "I implemented a ledger multiplexer in Go for Indy SDK (libindy).
+Be able to replace Indy ledger whit whatever verified data registry. Be able to
+have automatic ledger backups. Be able to remove the ledger's performance
+bottleneck with just a few simple and standard software engineering practices."
+author: Harri Lainio
+resources:
+- src: "**.{png,jpg}**"
+  title: "Image #:counter"
+---
+
+In this technical blog post, I'll explain how I implemented a plugin system into
+our Indy SDK Go wrapper and then extended it to work as a multiplexer. The plugin
+system allows us to use a key/value-based storage system instead of the normal
+Indy ledger. And multiplexing extended the functionality to use multiple data
+sources simultaneously and asynchronously. For instance, we can add a memory cache
+to help Indy ledger, which has been proving to make a considerable difference in
+a multi-tenant agency that can serve thousands of different wallet users simultaneously.
+
+{{< figure src="/blog/2022/09/27/ledger-multiplexer/Plugin.svg" >}}
```

Comment:
Image is not displaying, change 27 to 30

Reply:
There is a typo in the image "FINDY_LEGER" -> d is missing

Reply:
Cool, so cool! thank you so much!